### PR TITLE
RavenDB-16848 Highlight & Scroll to the relevant index on indexes vie…

### PR DESCRIPTION
**Pull request by Danielle9897**
…w after index creation + A similar fix for databases view

### CODE REVIEW

File: src/Raven.Studio/typescript/common/generalUtils.ts [hunk 1]
```diff
@@ -693,6 +693,23 @@ class genUtils {
 
         return result;
     }
+    
+    static scrollToElement(element: Element) {
+        const containerSelector = ".js-scroll-container";
```
Comment (ml054) [inline src/Raven.Studio/typescript/common/generalUtils.ts]: can we move this as second parameter with default value?

File: src/Raven.Studio/wwwroot/App/views/database/indexes/indexes.html [hunk 2]
```diff
@@ -183,7 +184,7 @@ <h2 class="on-base-background" data-bind="text: entityName, visible: !groupHidde
 </div>
 
 <script type="text/html" id="index-template">
-    <div class="padding padding-sm">
+    <div data-bind="attr: { class: 'padding padding-sm js-index-template ' + ($root.shouldHighlightIndex(name) ? 'blink-style-basic' : ''), id: 'id_' + name }">
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/database/indexes/indexes.html]: 1. can't we avoid having: id_ in id? It isn't meaningful.
2. What if index name contains characters which are invalid for id?

File: src/Raven.Studio/typescript/common/generalUtils.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/common/generalUtils.ts]: done

File: src/Raven.Studio/wwwroot/App/views/database/indexes/indexes.html (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/database/indexes/indexes.html]: rename ` id_` to `index_`

**Review by ml054 — APPROVED** [review]

**Pull request merged**